README.md

```diff
@@ -195,3 +200,32 @@ the builds. Build 306 was the last released with this process.
   `git push --tags`)
 
 * Send mail to python-win32
+
+</details>
+
+## Licenses
+
```

Comment:
I'm not sure this table is adding any value, and may even be detrimental as it becomes less accurate over time (eg, when new modules are added etc) - I think this section still makes sense and is useful if the table is omitted entirely?

Reply:
Sure, I can omit the table, it was still a valid exercise for me to triple check I didn't miss anything.